src/script/pages/app-index.ts

```diff
@@ -38,6 +38,22 @@ export class AppIndex extends LitElement {
           opacity: 1;
         }
       }
+      /* To handle sidebar & main */
```

Comment:
correct me if im wrong here, but the sidebar component is not actually in our HTML yet here right? Just the potential CSS for it? Im guessing this is intentional but wanted to check.

Reply:
Yes this was intentional to give an idea to reviewers how it's going to work

Reply:
Ok cool, thats what I assumed but just wanted to double check, thank you!